nucliadb_models/src/nucliadb_models/search.py

```diff
@@ -505,6 +521,11 @@ class SearchParamDefaults:
         title="Search features",
         description="List of search features to use. Each value corresponds to a lookup into on of the different indexes. `document`, `paragraph` and `vector` are deprecated, please use `fulltext`, `keyword` and `semantic` instead",  # noqa
     )
+    reranker = ParamDefault(
+        default=Reranker.MULTI_MATCH_BOOSTER,
+        title="Reranker",
+        description="Reranker let you specify which method you want to use to rerank your results at the end of retrieval",
```

Comment:
Maybe you could extend the description with a short sentence about each rerank option?

Reply:
I extended the information in the `Rerank` enum, I thought it'd be better, although not sure how is the end result in the docs